src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
 // write code here
+
+const table = document.querySelector('.field > tbody');
+const buttons = document.querySelector('.container');
+
+function modifyColumns(rezising) {
+  switch (rezising) {
+    case 'append-row':
+      table.appendChild(table.querySelector('tr').cloneNode(true));
+      break;
+    case 'remove-row':
+      table.deleteRow(table.rows.length - 1);
+      break;
+    case 'append-column':
+      [...table.rows].forEach((row) => {
+        row.appendChild(row.cells[row.cells.length - 1].cloneNode(true));
```

Comment:
Ensure that each row has at least one cell before attempting to clone it. If a row is empty, `row.cells[row.cells.length - 1]` will be `undefined`, causing an error.